libnetdata/inlined.h

```diff
@@ -510,6 +510,21 @@ static inline int read_single_signed_number_file(const char *filename, long long
     return 0;
 }
 
+static inline int read_single_base64_or_hex_number_file(const char *filename, unsigned long long *result) {
+    char buffer[30 + 1];
```

Comment:
Please, can you create a MACRO or to use an existing instead to use number `30`?

Reply:
I kept it the same as in the other `read_single_X` functions`.